doordeck-sdk/build.gradle.kts

```diff
@@ -378,14 +380,23 @@ tasks.register("csharpPack").configure {
     }
 }
 
+/**
+ * Generates the .nuspec file, which is required to create the .nupkg file
+ * needed for publishing a package to the NuGet repository.
+ */
+tasks.register("generateNuspecFile").configure {
+    // Define the output folder
+    val outputDir = file("$projectDir/build/bin/mingwX64/csharp")
+    // Create nuspec file
+    val nuspecFile = file("$outputDir/${nugetPublish.packageName}.nuspec")
+    nuspecFile.writeText(nuspecTemplate.trim())
+}
```

Comment:
Shouldn't this just be a copy task?

Reply:
It could be a copy task, I think, although I don't see the difference. Also, we are generating a new file rather than just copying it, so I'm not sure.